crates/viewer/re_space_view_spatial/src/visualizers/utilities/mod.rs

```diff
@@ -1,11 +1,13 @@
 pub mod entity_iterator;
 mod labels;
+mod proc_mesh_vis;
 mod spatial_view_visualizer;
 mod textured_rect;
 
 pub use labels::{
     process_labels, process_labels_2d, process_labels_3d, show_labels_fallback, LabeledBatch,
     UiLabel, UiLabelTarget,
 };
+pub(crate) use proc_mesh_vis::{ProcMeshBatch, ProcMeshVisBuffer};
```

Comment:
nit:
```suggestion
pub use proc_mesh_vis::{ProcMeshBatch, ProcMeshVisBuffer};
```

I found `pub(crate)` to be needlessly "infectious" which is why me (and Clement :smile:) are trying to use it more sparingly. It's most of the time uncessary since visibility is controlled on lib.rs level and otherwise just causes more things to require to be `pub(crate)`

(also applies to the respective added types)

Reply:
Done.

In case you're curious, my own habits in the area are to use a fair number of `pub use child_module::*`, and therefore to control the final external visibility on a more per-item basis, and to err on the side of *not* letting things possibly become actually public since — even lacking any concerns of API stability — that defeats dead code checking.